common/mariadb/templates/deployment.yaml

```diff
@@ -26,7 +26,9 @@ spec:
         prometheus.io/scrape: "true"
         prometheus.io/targets: {{ required ".Values.alerts.prometheus missing" .Values.alerts.prometheus | quote }}
 {{- end }}
-
+{{- if .Values.vpa.enabled }}
+        vpa-butler.cloud.sap/main-container: mariadb
+{{- end }}
```

Comment:
Why do have to enable it explicitly? Does it hurt if it's active by default?

I don't think `vpa.enabled` catches the meaning very good. I'd prefer something like `vpa.set_main_container`.

Reply:
only nova, neutron, keystone were part of the list. Other services were not.

Reply:
Doesn't answer the second question, but with the rename of the value, I'm fine.

Reply:
Should work, maybe make a toggle to remove the annotation instead?

Reply:
An Enable/Disable (toggle) option is more useful even if the default value is enabled or disable (true/false) than and option to remove if you don't want to enable it. In that case nobody won't forget to specify if they want to enable/disable it. Also it helps during templating.